apps/tasks.py

```diff
@@ -19,12 +19,7 @@
 
 
 def get_URI(parameters):
-    URI = (
-        "https://"
-        + parameters["release"]
-        + "."
-        + parameters["global"]["domain"]
-    )
+    URI = "https://" + parameters["release"] + "." + parameters["global"]["domain"]
```

Comment:
Will we be doing these fstrings?

Reply:
f"Yes, indeed" :-)   There are probably many such string concatenations throughout the code base that we can replace as we work on various parts.